test/ruby_lsp_rails/document_symbol_test.rb

```diff
@@ -287,6 +287,68 @@ class FooJob < ApplicationJob
         assert_empty(response[0].children)
       end
 
+      test "correctly handles validate method with all argument types" do
+        response = generate_document_symbols_for_source(<<~RUBY)
+          class FooModel < ApplicationRecord
+            validate "foo_arg", :bar_arg, -> () {}, Foo::BazClass.new("blah"), FooClass, Foo::BarClass
+          end
+        RUBY
+
+        assert_equal(1, response.size)
+        assert_equal("FooModel", response[0].name)
+        assert_equal(6, response[0].children.size)
+        assert_equal("validate(foo_arg)", response[0].children[0].name)
+        assert_equal("validate(bar_arg)", response[0].children[1].name)
+        assert_equal("validate(<anonymous>)", response[0].children[2].name)
+        assert_equal("validate(Foo::BazClass)", response[0].children[3].name)
+        assert_equal("validate(FooClass)", response[0].children[4].name)
+        assert_equal("validate(Foo::BarClass)", response[0].children[5].name)
+      end
+
+      test "correctly handles validates method with Prism::StringNode and Prism::SymbolNode argument types" do
```

Comment:
```suggestion
      test "correctly handles validates method with string and symbol argument types" do
```
To avoid the being overly specific about the implementation.

Reply:
This is also how the other tests for callbacks are structured, so I pushed a commit to fix this for them all.